script/build.js

```diff
@@ -205,6 +214,6 @@ if (options.watch) {
 }
 
 smith.build(function(err) {
-    if (err) throw err;
-    console.log('Build finished!');
-  });
+  if (err) throw err;
+  console.log('Metalsmith build finished!  Now starting webpack...');
```

Comment:
Actually, the behavior is slightly different for `--watch`.  Webpack seems to run afterwards.